react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AppAccordion from './accordion.jsx';
+
+ReactDOM.render(
+  <AppAccordion />,
```

Comment:
Your array of items should be defined here, and passed to the Accordion component via props.